WebAPI/WebAPI/DTOs/EmployeeDTO.cs

```diff
@@ -6,18 +6,23 @@ namespace WebAPI.DTOs
 {
     public class EmployeeDTO
     {
-        public int Id { get; set; }
+        public Guid ID { get; set; }
         public double SSN { get; set; }
-        public string FullName { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
         public string Address { get; set; }
         public string PhoneNumber { get; set; }
         public int Gender { get; set; }
-        public string Nationality { get; set; }
+        public string Headline { get; set; }
+        public string Country { get; set; }
         public DateOnly BirthDate { get; set; }
-        public DateOnly ContractDate { get; set; }
-        public decimal BaseSalary { get; set; }
+        public DateOnly JoinDate { get; set; }
+        public decimal FixedSalary { get; set; }
+        public decimal GrossSalary { get; set; }
         public TimeOnly Arrival { get; set; }
```

Comment:
Delete Arrival from Employee and EmployeeDTO, because we have the separated shift module.